client/src/App.tsx

```diff
@@ -1,15 +1,31 @@
-import './App.css';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
 import BookingPage from './pages/BookingPage';
 import CheckinPage from './pages/CheckinPage';
-import {TopBar} from './components/TopBar';
-import {Route} from 'react-router-dom'
+import { TopBar } from './components/TopBar';
+import { Route } from 'react-router-dom'
+import HomePage from './pages/HomePage';
+import { CssBaseline } from '@mui/material';
 
 function App() {
+  const theme = createTheme({
+    palette: {
+      primary: {
+        main: '#ff9800',
```

Comment:
You can create a separate file theme.ts that exports the theme variable. 
Don't use hex values for colors, but import colors from "@mui/material"